gradle/libs.versions.toml

```diff
@@ -33,16 +36,33 @@ androidx-constraintlayout = "androidx.constraintlayout:constraintlayout:2.1.0-be
 androidx-coreKtx = "androidx.core:core-ktx:1.6.0-beta01"
 androidx-documentfile = "androidx.documentfile:documentfile:1.0.1"
 androidx-fragmentKtx = "androidx.fragment:fragment-ktx:1.4.0-alpha01"
-androidx-lifecycle_common = { module = "androidx.lifecycle:lifecycle-common-java8", version.ref="lifecycle" }
-androidx-lifecycle_livedataKtx = { module = "androidx.lifecycle:lifecycle-livedata-ktx", version.ref="lifecycle" }
-androidx-lifecycle_viewmodelKtx = { module = "androidx.lifecycle:lifecycle-viewmodel-ktx", version.ref="lifecycle" }
+androidx-hilt-compose = "androidx.hilt:hilt-navigation-compose:1.0.0-alpha02"
+androidx-lifecycle-common = { module = "androidx.lifecycle:lifecycle-common-java8", version.ref="lifecycle" }
+androidx-lifecycle-livedataKtx = { module = "androidx.lifecycle:lifecycle-livedata-ktx", version.ref="lifecycle" }
+androidx-lifecycle-viewmodelKtx = { module = "androidx.lifecycle:lifecycle-viewmodel-ktx", version.ref="lifecycle" }
+androidx-lifecycle-viewmodel-compose = "androidx.lifecycle:lifecycle-viewmodel-compose:1.0.0-alpha05"
 androidx-material = "com.google.android.material:material:1.4.0-beta01"
 androidx-preference = "androidx.preference:preference:1.1.1"
 androidx-recyclerview = "androidx.recyclerview:recyclerview:1.2.0"
 androidx-recyclerviewSelection = "androidx.recyclerview:recyclerview-selection:1.2.0-alpha01"
 androidx-security = "androidx.security:security-crypto:1.1.0-alpha03"
 androidx-swiperefreshlayout = "androidx.swiperefreshlayout:swiperefreshlayout:1.2.0-alpha01"
 
+# Compose dependencies
+compose-animation-animation = { module = "androidx.compose.animation:animation", version.ref = "compose" }
+compose-compiler-compiler = { module = "androidx.compose.compiler:compiler", version.ref = "compose" }
+compose-foundation-foundation = { module = "androidx.compose.foundation:foundation", version.ref = "compose" }
+compose-foundation-layout = { module = "androidx.compose.foundation:foundation-layout", version.ref = "compose" }
+compose-material-iconsext = { module = "androidx.compose.material:material-icons-extended", version.ref = "compose" }
```

Comment:
We're never going to use this, the size penalty is exorbitant

Reply:
That won't be an issue in release builds since R8 should remove most of the classes from there. I say let's keep the dependency there and we can decide about this later? or if you're 100% sure, I can just remove this now.

Reply:
It'll only serve to fragment icon resources between drawable XMLs and code, we're not using it.